packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -127,7 +127,10 @@ export interface AnchorEvents {
 	 *
 	 * Compare to {@link AnchorEvents.childrenChanged} which is emitted in the middle of the batch/delta-visit.
 	 */
-	childrenChangedAfterBatch(anchor: AnchorNode): void;
+	childrenChangedAfterBatch(arg: {
```

Comment:
I realize these eventing methods aren't part of the public API, so this probably doesn't matter too much, but (for consistency) should we go ahead and update all of these methods to take a property bag containing the anchor?